include/deal.II/lac/lapack_templates.h

```diff
@@ -20,6 +20,8 @@
 #include <deal.II/base/config.h>
 #include <deal.II/lac/lapack_support.h>
 
+#include <cfenv>
```

Comment:
You will have to guard this `#include` file with `#ifdef DEAL_II_HAVE_FP_EXCEPTIONS  ...   #endif`. The same is true for all of the calls below to any of the functions you add.

Reply:
I added the guards. I'm wondering whether I should add an `#else` if the condition is not fulfilled?